### ISSUE TITLE
[4.0] Logo moves (a11y)

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#27858]
### Expected result
Logo is loaded where it will appear

### Actual result
Logo is loaded in the middle and then moves to the left. 
![logo](https://user-images.githubusercontent.com/1296369/74091056-15926480-4aab-11ea-9210-cd0e826a867c.gif)

### Additional comments
Its something in the scss perhaps display: inline-block on the logo but I can't quite work it out

### Additional comments accessibility
Can't find the actual WCAG rule number but its an a11y fail to have something that flashes on the page as it distracts the user

**Comment by MostafaHamedAbdelmasoud** [comment]
Is issue fixed now?
Can I work on it?

**Comment by brianteeman** [comment]
@MostafaHamedAbdelmasoud you don't need to ask

**Comment by brianteeman** [comment]
Further investigation shows that the logo is moved by the javascript here but I cannot work out why it is being triggered and why it is not left aligned without the javascript.

https://github.com/joomla/joomla-cms/blob/69052cad4e38b58c8201e5f39257a4bd1dc19681/build/media_source/templates/atum/js/template.es6.js#L64

**Comment by brianteeman** [comment]
closed see #27880
